scripts/check_md5sums.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+set -e
+
+if [ $# -ne 1 ] || ! { [ "$1" = "zson" ] || [ "$1" = "bzson" ]; }; then
+  echo 'Must specify output type to be cheecked: "zson" or "bzson"'
+  exit 1
+fi
+
+ZNG_TYPE="$1"
+REPO_DIR="$(cd "$(dirname "$0")" && pwd)/.."
+cd "$REPO_DIR"
+TMPFILE=$(mktemp)
+
+for FILE in "$REPO_DIR"/"$ZNG_TYPE"/*
+do
+  COMPARE_TO="$(basename "$FILE")"
+  ZPATH=${COMPARE_TO/.${ZNG_TYPE}.gz/}
+  echo -n "${ZPATH}:" | tee -a "$TMPFILE"
+  gzcat zeek-default/"$ZPATH".log.gz | zq -f "$ZNG_TYPE" - | md5sum | awk '{ print $1 }' | tee -a "$TMPFILE"
+done
+
+echo
+echo "diff'ing current \"zq -f $ZNG_TYPE\" output hashes vs. committed hashes:"
```

Comment:
It would be nice to compare file by file rather than hashing everything first and then comparing. but that's a feature request rather than a script change request :)

Reply:
Yeah, fair enough. I'm figuring I'll start simple and see what we find before adding more features. Thankfully the data set is so small that it can do the full run through of the hashes in just a few seconds.

Not sure if you were maybe thinking of actually walking the user through the _diffs_ file-by-file. That's something I thought of as well, but decided against at this stage.

Reply:
Yes that was the motivation. But as I said, feature request!